blocks/init/src/Blocks/components/copyright/manifest.json

```diff
@@ -5,7 +5,7 @@
 	"example": {
 		"attributes": {
 			"copyrightBy": "Eightshift",
-			"copyrightYear": "2020",
+			"copyrightYear": "2021",
```

Comment:
When in Rome... Can this be removed from the manifest? And just rendered out in PHP/JS so that we don't need to change the year manually?

Reply:
this is just an example for storybook. No need to change this we can put 1970 just for fun :D

Reply:
The default footer renders the current year, I'll remove the example attr, but leave the attribute though, some websites like to have a static (C) year.